GVRf/Framework/framework/src/main/java/org/gearvrf/GVRRenderTarget.java

```diff
@@ -138,9 +138,22 @@ public GVRRenderTexture getTexture()
         return mTexture;
     }
 
-    public void onDrawFrame(float frameTime)
+    private GVRDrawFrameListener  mDrawListener = new GVRDrawFrameListener()
     {
-        getGVRContext().getActivity().getViewManager().cullAndRender(this, mScene);
+        public void onDrawFrame(float frameTime)
+        {
+            getGVRContext().getActivity().getViewManager().cullAndRender(GVRRenderTarget.this, mScene);
+        }
+    };
+
+    public void startRendering()
```

Comment:
Public APIs need JavaDoc.

Reply:
This is the reason why blurfilter and rendertotexture not working. With this new API, we need to call startRendering() from app who create render target. Will discuss with @NolaDonato how it should be done.